nix/scripts/node_modules.sh

```diff
@@ -68,7 +68,7 @@ findFilesNewerThan() {
       \ # related code https://github.com/lugg/react-native-config/blob/v1.5.0/react-native-config.podspec#L52 \
       -not -path "*/node_modules/react-native-config/ios/ReactNativeConfig/GeneratedDotEnv.m" \
       \ # generated at runtime by react-native from v0.73.x onwards \
-      -not -path "*/node_modules/react-native/React/Fabric/*" \
+      -not -path "*/node_modules/react-native/React/Fabric/RCTThirdPartyFabricComponentsProvider.mm" \
```

Comment:
Had to make this change because I was reproducing https://github.com/status-im/status-mobile/issues/18869 again